webapp/store/views.py

```diff
@@ -14,9 +14,9 @@ def store():
     return render_template('store/store.html')
 
 
-@jaasstore.route('/q/<path:query>/')
-def search(query):
-    query = query.replace('/', ' ')
+@jaasstore.route('/q/')
+def search():
+    query = request.args.get('search').replace('/', ' ')
```

Comment:
If we're changing the scheme anyway, it's worth briefly considering what we want the URL scheme to be for search.

Unless you have other ideas I'd suggest following the scheme we use on ubuntu.com - `https://www.ubuntu.com/search?q=hello+world` - so `@jaasstore.route('/search')` & `request.args.get('q')`?

The ubuntu.com scheme follows advice in [Apigee's "Web API design"](https://pages.apigee.com/rs/apigee/images/api-design-ebook-2012-03.pdf), which contains sensible practical advice for designing URLs, which I tend to agree with in every case.

Unless we think there's a better option?